.vscode/extensions.json

```diff
@@ -0,0 +1,18 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		"dbaeumer.vscode-eslint",
+    "kisstkondoros.vscode-codemetrics",
+    "EditorConfig.EditorConfig",
+    "PhilHindle.errorlens",
+    "wix.vscode-import-cost",
+    "syler.sass-indented"
```

Comment:
Maybe we need ESLint on these files too... Just kidding. Could we format this file correctly though. I think we have a mixture of tabs and spaces!